packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1343,8 +1343,8 @@ ensureIndex(Posts,
 Posts.addView("reviewFinalVoting", (terms: PostsViewTerms) => {
   return {
     selector: {
-      reviewCount: { $gte: 1},
-      positiveReviewVoteCount: { $gte: 1 },
+      reviewCount: { $gte: 1 },
+      positiveReviewVoteCount: { $gte: 1 }, // TODO: Ray thinks next year this should change to "has at least 4 points"
```

Comment:
Why not this year?

Reply:
I went back and forth a bit, and decided the benefits of filtering them out didn't outweigh the integrity-hit to changing the requirements last minute. (I'm not that confident in that, but, currently think the effect size is smallish either way)

Reply:
Yeah, makes sense